src/navigator/ProfileMenu.tsx

```diff
@@ -0,0 +1,173 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SafeAreaView, StyleSheet, Text, View } from 'react-native'
+import deviceInfoModule from 'react-native-device-info'
+import { useSelector } from 'react-redux'
+import { defaultCountryCodeSelector, e164NumberSelector, nameSelector } from 'src/account/selectors'
+import { phoneNumberVerifiedSelector } from 'src/app/selectors'
+import AccountNumber from 'src/components/AccountNumber'
+import ContactCircleSelf from 'src/components/ContactCircleSelf'
+import PhoneNumberWithFlag from 'src/components/PhoneNumberWithFlag'
+import Touchable from 'src/components/Touchable'
+import Times from 'src/icons/Times'
+import Help from 'src/icons/navigator/Help'
+import { Invite } from 'src/icons/navigator/Invite'
+import Settings from 'src/icons/navigator/Settings'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+import RewardsPill from 'src/navigator/RewardsPill'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { NETWORK_NAMES } from 'src/shared/conts'
+import colors, { Colors } from 'src/styles/colors'
+import fontStyles, { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import { getSupportedNetworkIdsForTokenBalances } from 'src/tokens/utils'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.ProfileMenu>
+
+export default function ProfileMenu({ route }: Props) {
+  const { t } = useTranslation()
+  const displayName = useSelector(nameSelector)
+  const e164PhoneNumber = useSelector(e164NumberSelector)
+  const defaultCountryCode = useSelector(defaultCountryCodeSelector)
+  const account = useSelector(currentAccountSelector)
+  const appVersion = deviceInfoModule.getVersion()
+  const phoneNumberVerified = useSelector(phoneNumberVerifiedSelector)
+  const networks = getSupportedNetworkIdsForTokenBalances()
+  const networkNames = networks.map((network) => NETWORK_NAMES[network])
+  return (
+    <SafeAreaView>
+      <View style={styles.top}>
+        <View style={styles.header} testID="ProfileMenu/Header">
+          <Touchable
+            onPress={navigateBack}
+            borderless={true}
+            hitSlop={variables.iconHitslop}
+            testID="InviteModalCloseButton"
+          >
+            <Times />
+          </Touchable>
+          <RewardsPill useValoraIcon={true} />
+        </View>
```

Comment:
Can we set these on the standard header instead of the custom header?

Reply:
@satish-ravi I changed it to using the standard header, however now the X seems too far to the left as compared to the designs, but I'm not sure how to change it since it is the `headerLeft` item now?